SFSymbol2/SFSymbol2.0.swift

```diff
@@ -0,0 +1,811 @@
+    case square3stack3dbottomfill = "square3Stack3DBottomFill"
+    case cylindersplit1x2 = "cylinderSplit1X2"
+    case cylindersplit1x2fill = "cylinderSplit1X2Fill"
+    case chartbarxaxis = "chartBarXaxis"
+    case dotsquareshapesplit2x2 = "dotSquareshapeSplit2X2"
+    case squareshapesplit2x2 = "squareshapeSplit2X2"
+    case squareshapesplit3x3 = "squareshapeSplit3X3"
+    case waveformpathecgrectangle = "waveformPathEcgRectangle"
+    case waveformpathecgrectanglefill = "waveformPathEcgRectangleFill"
+    case staroflifecircle = "staroflifeCircle"
+    case staroflifecirclefill = "staroflifeCircleFill"
+    case simcard = "simcard"
+    case simcardfill = "simcardFill"
+    case simcard2 = "simcard2"
+    case simcard2fill = "simcard2Fill"
+    case sdcard = "sdcard"
+    case sdcardfill = "sdcardFill"
+    case touchid = "touchid"
+    case bonjour = "bonjour"
+    case atom = "atom"
+    case scalemass = "scalemass"
+    case scalemassfill = "scalemassFill"
+    case headphonescircle = "headphonesCircle"
+    case headphonescirclefill = "headphonesCircleFill"
+    case giftcircle = "giftCircle"
+    case giftcirclefill = "giftCircleFill"
+    case arrowuprightapp = "arrowUpRightApp"
+    case arrowuprightappfill = "arrowUpRightAppFill"
+    case arrowdownapp = "arrowDownApp"
+    case arrowdownappfill = "arrowDownAppFill"
+    case airplanecircle = "airplaneCircle"
+    case airplanecirclefill = "airplaneCircleFill"
+    case hourglassbadgeplus = "hourglassBadgePlus"
+    case banknote = "banknote"
+    case banknotefill = "banknoteFill"
+    case paragraphsign = "paragraphsign"
+    case camerafilters = "cameraFilters"
+    case arrowleftandrightrighttriangleleftrighttriangleright = "arrowLeftAndRightRighttriangleLeftRighttriangleRight"
+    case arrowleftandrightrighttriangleleftrighttrianglerightfill = "arrowLeftAndRightRighttriangleLeftRighttriangleRightFill"
+    case arrowupanddownrighttriangleuprighttriangledown = "arrowUpAndDownRighttriangleUpRighttriangleDown"
+    case arrowupanddownrighttriangleupfillrighttriangledownfill = "arrowUpAndDownRighttriangleUpFillRighttriangleDownFill"
+    case arrowtriangleleftandlineverticalandarrowtriangleright = "arrowtriangleLeftAndLineVerticalAndArrowtriangleRight"
+    case arrowtriangleleftfillandlineverticalandarrowtrianglerightfill = "arrowtriangleLeftFillAndLineVerticalAndArrowtriangleRightFill"
+    case arrowtrianglerightandlineverticalandarrowtriangleleft = "arrowtriangleRightAndLineVerticalAndArrowtriangleLeft"
+    case arrowtrianglerightfillandlineverticalandarrowtriangleleftfill = "arrowtriangleRightFillAndLineVerticalAndArrowtriangleLeftFill"
+    case lifepreserver = "lifepreserver"
+    case lifepreserverfill = "lifepreserverFill"
+    case binoculars = "binoculars"
+    case binocularsfill = "binocularsFill"
+    case battery100bolt = "battery100Bolt"
+    case minusplusbatteryblock = "minusPlusBatteryblock"
+    case minusplusbatteryblockfill = "minusPlusBatteryblockFill"
```

Comment:
The names of the `case` statements so far use snakeCase so for example `case minusplusbatteryblockfill = "minusPlusBatteryblockFill"` should be `case minusPlusBatteryBlockFill = "minusPlusBatteryblockFill"`